src/modules/70-pipeline/pages/inputSet-list/InputSetListView.tsx

```diff
@@ -240,7 +243,8 @@ const RenderColumnActions: Renderer<CellProps<InputSetLocal>> = ({ row, column }
 
   return (
     <RbacButton
-      disabled={!(column as any)?.pipelineHasRuntimeInputs}
+      disabled={!(column as any)?.pipelineHasRuntimeInputs || isPipelineInvalid}
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9275140)
```suggestion
      disabled={!(column as unknown)?.pipelineHasRuntimeInputs || isPipelineInvalid}
```